kong/dao/migrations/helpers.lua

```diff
@@ -95,5 +99,205 @@ function _M.plugin_config_iterator(dao, plugin_name)
   end
 end
 
+_M.cassandra = {}
+
+--[[
+Copy all records from the table defined by source_table_def into
+destination_table_def. Both table_defs have the following structure
+  { name    = "ssl_certificates",
+    columns = {
+      id         = "uuid",
+      cert       = "text",
+      key        = "text",
+      created_at = "timestamp",
+    },
+    partition_keys = { "id" },
+  }
+
+columns_to_copy is a hash-like table.
+* Each key must be a string D representing a column in the destination table.
+* If the value is a string S, then the value of the S column in the source
+  table will be assigned to the D column in destination.
+* If the value is a function, then the result of executing it will be assigned
+  to the D column.
+Example:
+  {
+    partition  = function() return cassandra.text("certificates") end,
+    id         = "id",
+    cert       = "cert",
+    key        = "key",
+   created_at = "created_at",
+  }
+
+The function takes the "source row" as parameter, so it could be used to do things
+like merging two fields together into one, or putting a string in uppercase.
+--]]
+function _M.cassandra.copy_records(dao,
+                                   source_table_def,
+                                   destination_table_def,
+                                   columns_to_copy)
+
+  local coordinator = dao.db:get_coordinator()
```

Comment:
I think we need to catch the error potentially returned by `get_coordinator()` here (which is susceptible to a bug, as noticed by @salazar, but this is out of scope for now)

Reply:
Ok, #3374 will now help us here :)